charts/perses/templates/statefulset.yaml

```diff
@@ -77,7 +77,7 @@ spec:
             containerPort: {{ .Values.service.targetPort }}
         readinessProbe:
           httpGet:
-            path: /metrics
+            path: {{ .Values.config.api_prefix}}/metrics
```

Comment:
```suggestion
            path: {{ .Values.config.api_prefix }}/metrics
```

Reply:
fixed in https://github.com/perses/helm-charts/pull/46/commits/e0d858e009bf9d77ec516035d0c065cce238bbd1